### PR TITLE
Centered the cropped image

### LINKED ISSUE #698: Center Cropped Image
### Please describe the problem (or idea)

After an image is cropped it is aligned to the top left corner. Instead, it can be centered.

> What happened just before the problem occurred? Or what problem could this idea solve?

screenshot from 2019-01-22 11-32-01

### Possible FTO

This is a potential `first-timers-only` issue. If you are new to the community, then you can reformat this issue into a first timer only issue and mention `@publiclab/is-reviewers` to add the label. If you have contributed to Image Sequencer before, then you can resolve this one as a second-time contribution.

##### HTML (this is where the img element is)

https://github.com/publiclab/image-sequencer/blob/17d79c4b44c5e5b8f7f8b4574a783d5b87cdf871/examples/lib/defaultHtmlStepUi.js#L33-L36

#### Possible fixes.

A bootstrap class or some CSS can be used to center the image

### Please show us where to look

http://sequencer.publiclab.org...

### What's your PublicLab.org username?

> This can help us diagnose the issue:

### Browser, version, and operating system

> Many bugs are related to these -- please help us track it down and reproduce what you're seeing!

****

## Thank you!

Your help makes Public Lab better! We *deeply* appreciate your helping refine and improve this site.

To learn how to write really great issues, which increases the chances they'll be resolved, see:

https://publiclab.org/wiki/developers#Contributing+for+non-coders

### PR DESCRIPTION
**Pull request by harsh1599**
Fixes #698 

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

* [ ] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `npm test`
* [ ] code is in uniquely-named feature branch and has no merge conflicts
* [ ] PR is descriptively titled
* [ ] ask `@publiclab/reviewers` and `@publiclab/is-reviewers` for help, in a comment below

> We're happy to help you get this ready -- don't be afraid to ask for help, and **don't be discouraged** if your tests fail at first!

If tests do fail, click on the red `X` to learn why by reading the logs.

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/contributing-to-public-lab-software

Thanks!

### CODE REVIEW

File: examples/lib/defaultHtmlStepUi.js [hunk 1]
```diff
@@ -34,9 +34,11 @@ function DefaultHtmlStepUi(_sequencer, options) {
       '</i></p></div>\
     </div>\
     </form>\
-    <div class="col-md-8 cal">\
-    <div class="load" style="display:none;"><i class="fa fa-circle-o-notch fa-spin"></i></div>\
-    <a><img alt="" style="max-width=100%" class="img-thumbnail step-thumbnail"/></a>\
+    <div class="col-md-8 cal crop-column">\
+      <div class="load" style="display:none;"><i class="fa fa-circle-o-notch fa-spin"></i></div>\
+      <div class = "crop-image">
```
Comment (HarshKhandeparkar) [inline examples/lib/defaultHtmlStepUi.js]: Please don't keep any gaps in between the equal to sign and the rest of the code.

**Comment by HarshKhandeparkar** [comment]
@harsh1599 please change the name of the PR to describe the problem which is solved.

Also please remove the `#[add issue number here.]` and replace it with a link to the issue. Thanks!

File: examples/lib/defaultHtmlStepUi.js (hunk 1 above)
Comment (harsh1599) [reply examples/lib/defaultHtmlStepUi.js]: Sorry..I have a habit of leaving spaces. I've made the changes

File: examples/lib/defaultHtmlStepUi.js [hunk 2]
```diff
@@ -34,9 +34,11 @@ function DefaultHtmlStepUi(_sequencer, options) {
       '</i></p></div>\
     </div>\
     </form>\
-    <div class="col-md-8 cal">\
-    <div class="load" style="display:none;"><i class="fa fa-circle-o-notch fa-spin"></i></div>\
-    <a><img alt="" style="max-width=100%" class="img-thumbnail step-thumbnail"/></a>\
+    <div class="col-md-8 cal crop-column">\
+      <div class="load" style="display:none;"><i class="fa fa-circle-o-notch fa-spin"></i></div>\
+      <div class="crop-image">
```
Comment (HarshKhandeparkar) [inline examples/lib/defaultHtmlStepUi.js]: Oh I know why. Please add backslash at the end of the line. You have to do that as this is js and not HTML. Thanks.

**Comment by harsh1599** [comment]
Sorry..it took a lot of time since I suck at css

**Comment by HarshKhandeparkar** [comment]
@publiclab/is-reviewers

**Comment by HarshKhandeparkar** [comment]
> Sorry..it took a lot of time since I suck at css

No problem. You will only gain experience from this.

**Comment by HarshKhandeparkar** [comment]
Even I suck at CSS 😜😜

**Comment by HarshKhandeparkar** [comment]
@harsh1599 please provide screenshots for different screen sizes and different crop sizes.

File: examples/demo.css [hunk 3]
```diff
@@ -245,3 +245,16 @@ a.name-header{
   text-decoration: none;
   color: #445;
 }
+
+.crop-column{
+    height:648.19px; 
+    display:flex; 
+    flex-direction:row; 
+    align-items:center
```
Comment (HarshKhandeparkar) [inline examples/demo.css]: Please add semi colons

File: examples/demo.css [hunk 4]
```diff
@@ -245,3 +245,16 @@ a.name-header{
   text-decoration: none;
   color: #445;
 }
+
+.crop-column{
+    height:648.19px;
```
Comment (HarshKhandeparkar) [inline examples/demo.css]: Please don't use such heights, you can use '%' or 'rem' etc.

**Comment by harsh1599** [comment]
Screenshots for different screen sizes:
screenshot from 2019-01-24 17-29-13
screenshot from 2019-01-24 17-29-19
screenshot from 2019-01-24 17-29-25
screenshot from 2019-01-24 17-29-45

**Comment by harsh1599** [comment]
for different crop sizes:
(Kindly suggest a critical size if possible):
screenshot from 2019-01-24 17-33-25
screenshot from 2019-01-24 17-34-14
screenshot from 2019-01-24 17-34-22
screenshot from 2019-01-24 17-35-13
screenshot from 2019-01-24 17-35-25

File: examples/lib/defaultHtmlStepUi.js [hunk 5]
```diff
@@ -34,9 +34,11 @@ function DefaultHtmlStepUi(_sequencer, options) {
       '</i></p></div>\
     </div>\
     </form>\
-    <div class="col-md-8 cal">\
-    <div class="load" style="display:none;"><i class="fa fa-circle-o-notch fa-spin"></i></div>\
-    <a><img alt="" style="max-width=100%" class="img-thumbnail step-thumbnail"/></a>\
+    <div class="col-md-8 cal step-column">\
+      <div class="load" style="display:none;"><i class="fa fa-circle-o-notch fa-spin"></i></div>\
+      <div class="step-image">\
+        <a><img alt="" style="max-width=100%" class="img-thumbnail step-thumbnail"/></a>\
```
[the remark this answers is not in the document]
Comment (harsh1599) [reply examples/lib/defaultHtmlStepUi.js]: Nothing changes..and I don't think it's required..so shall I remove it?

[the remark this answers is not in the document]
Comment (HarshKhandeparkar) [reply examples/lib/defaultHtmlStepUi.js]: Yes remove it

**Comment by jywarren** [comment]
This looks really nice! Thanks!!!

**Comment by jywarren** [comment]
Great work!!!

**Pull request merged**